### PR TITLE
Actor relation CSV import fix, refs #13303

### PR DESCRIPTION
**Pull request by mcantelon**
Changed actor relation CSV import so import doesn't halt, and instead
skips the current row, if a non-existent relation type is encountered.

### CODE REVIEW

File: lib/task/import/csvAuthorityRecordRelationImportTask.class.php [hunk 1]
```diff
@@ -113,14 +113,15 @@ private function import($filepath, $sourceName, $indexDuringImport = false)
...
+        $relationTypeId = array_search_case_insensitive(
           $self->columnValue('relationType'),
           $self->status['actorRelationTypes'][$self->columnValue('culture')]
         );
 
         if (!$relationTypeId)
         {
-          throw new sfException(sprintf('Unknown relationship type %s:', $self->columnValue('relationType')));
+          $error = sprintf('Unknown relationship type "%s"... skipping row.', $self->columnValue('relationType'));
+          print $self->logError($error);
```
Comment (jraddaoui) [inline lib/task/import/csvAuthorityRecordRelationImportTask.class.php:124]: Is `print` really needed in here and in L137?

**Review by jraddaoui — APPROVED** [review]
Looks good @mcantelon!

File: lib/task/import/csvAuthorityRecordRelationImportTask.class.php (hunk 1 above)
Comment (mcantelon) [reply lib/task/import/csvAuthorityRecordRelationImportTask.class.php:124]: Yeah print or echo are needed as the log error function returns the argument prepended with the current CSV row number of the import (kind of unintuitive, alas).

**Comment by mcantelon** [comment]
Thanks @jraddaoui !

**Pull request merged**